pkg/ccl/storageccl/bench_test.go

```diff
@@ -139,21 +139,25 @@ func BenchmarkImport(b *testing.B) {
 	tempDir, dirCleanupFn := testutils.TempDir(b)
 	defer dirCleanupFn()
 
+	args := base.TestClusterArgs{}
+	args.ServerArgs.ExternalIODir = tempDir
+
 	for _, numEntries := range []int{1, 100, 10000, 300000} {
 		b.Run(fmt.Sprintf("numEntries=%d", numEntries), func(b *testing.B) {
 			bankData := sampledataccl.BankRows(numEntries)
-			backupDir := filepath.Join(tempDir, strconv.Itoa(numEntries))
+			subdir := strconv.Itoa(numEntries)
+			backupDir := filepath.Join(tempDir, subdir)
 			backup, err := sampledataccl.ToBackup(b, bankData, backupDir)
 			if err != nil {
 				b.Fatalf("%+v", err)
 			}
-			storage, err := storageccl.ExportStorageConfFromURI(`nodelocal://` + backupDir)
+			storage, err := storageccl.ExportStorageConfFromURI(`nodelocal:///` + subdir)
```

Comment:
I assume you're asking for my review because my dependency bump broke this.

Can you provide some context, preferable in the commit message, explaining this solution? Looking at this code, it looks like a typo - i.e. why would the protocol identifier be followed by three slashes?

Perhaps this would be clearer if a `(*url.URL).String` was used in place of string concatenation.

Reply:
Err, didn't mean to ask for your review at all. Sorry about that.

Reply:
:+1:

Nevertheless, looking like you want `path.Abs` here? At least, that's what https://github.com/cockroachdb/cockroach/blob/e9f116061b5406c24eed88b415b8c90da42e5fbd/pkg/ccl/storageccl/export_storage.go#L214-L222 seems to suggest.